tests/testthat/test-translate-hive.R

```diff
@@ -0,0 +1,150 @@
+context("translate-Hive")
+
+# Hive base_scalar conversions -----------------------------------------
+
+test_that("as.numeric() translates to DOUBLE ", {
```

Comment:
Since you now have one function that generates all the casts, you can just test that function and delete all these tests.